setup.cfg

```diff
@@ -84,7 +84,6 @@ testpaths =
 addopts =
   --verbose
   --emoji
-  --flake8
```

Comment:
OK, so I don't really have a problem with relaxing the standards a bit, but if we remove flake8 then we should at least include  `--pycodestyle` and `--pydocstyle`.

Reply:
actually pycodestyle would be in conflict with black, there is pytest-black however and its --black option. 
I would not add it now, but rather with the black formatting PR.